src/complex/Utilities/Math/GeometryMath.hpp

```diff
@@ -23,12 +24,18 @@ namespace GeometryMath
  * vector. The vectors are assumed to cross at (0,0,0).
  * @param a
  * @param b
- * @return float
+ * @return T
  */
 template <typename T>
-T CosThetaBetweenVectors(const complex::Point3D<T>& a, const complex::Point3D<T>& b)
+inline T CosThetaBetweenVectors(const complex::Point3D<T>& a, const complex::Point3D<T>& b)
 {
-  throw std::runtime_error("");
+  T norm1 = sqrt(a.sumOfSquares());
+  T norm2 = sqrt(b.sumOfSquares());
```

Comment:
```suggestion
  T norm1 = std::sqrt(a.sumOfSquares());
  T norm2 = std::sqrt(b.sumOfSquares());
```
All `<cmath>` functions should have the `std::` namespace prefixed.